### DISCUSSION TITLE
作者您好，请问可以更新一下弹幕下载的功能吗

### DISCUSSION THREAD

**Discussion by renceing** [yaobiao131/downkyicore#60] [General]
一直以来是downkyi的忠实用户，其他功能都非常好用，只是弹幕相较一些脚本（比如”bilibili视频下载“)弹幕最终下载的条数少很多，一般会少1/2到2/3左右，作者有空闲时能否考虑补全此方面功能，感激不尽！

**Comment by yaobiao131** [comment]
最新提交已经修复此问题。下个版本更新就修复

**Reply by yaobiao131** [reply]
https://github.com/yaobiao131/downkyicore/commit/4243f50866609640c362a695da16d5fbfe958d39

**Reply by renceing** [reply]
感谢！辛苦了♥
